fuel-core/src/config/chain_config.rs

```diff
@@ -21,12 +19,22 @@ pub const TESTNET_INITIAL_BALANCE: u64 = 10_000_000;
 pub struct ChainConfig {
```

Comment:
I'm thinking we should maybe rename this to chainspec and keep it in a different module namespace. It's more focused on genesis config, while the other config concerns are primarily runtime configurable params for tuning various modules within the client.

Reply:
Yeah, `Chainspec` is a lot nicer.